src/AppIndicator.py

```diff
@@ -41,6 +45,22 @@ def __init__(self, TEST_MODE: bool = False):
         self.daemon.setDaemon(True)
         self.daemon.start()
 
+    def __get_icon_path(self):
```

Comment:
We are parsing configs also [here](https://github.com/maateen/battery-monitor/blob/master/battery_monitor/SettingsWindow.py#L112) in `SettingsWindow.py`. That isn't anymore feasible to me.

Can we parse the configs at `config.py` and then simply import Global Variables where necessary?

Reply:
Sure, I think that should work. However, when we do that wouldn't we mix up the application config and user config? Idk, it probably would be okay, but we could also add a new class which only parses the user config into an array or multiple Global Variables. Then

Reply:
> but we could also add a new class which only parses the user config into an array or multiple Global Variables.

It looks good to me.